src/DssSpell.sol

```diff
@@ -52,157 +67,124 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE      = ;
+    uint256 internal constant ZERO_PT_TWO_FIVE_PCT_RATE = 1000000000079175551708715274;
+
+    uint256 internal constant MILLION = 10 ** 6;
+    // uint256 internal constant RAY  = 10 ** 27;
+    uint256 internal constant WAD     = 10 ** 18;
+
+    ChainLogLike internal immutable CHAINLOG    = ChainLogLike(DssExecLib.getChangelogAddress("CHANGELOG"));
+    VatLike      internal immutable VAT         = VatLike(DssExecLib.vat());
+    address      internal immutable DOG         = DssExecLib.dog();
```

Comment:
I dislike the mixing of the `VAT` as a `VatLike` with the `DOG` as an `address`.  I think if we want to use a `VatLike` we should change the case on `VAT` to `vat`.

Reply:
its camps because its immutable i think.  we need `VatLike` to do line changes, but we just need the `DOG` address

Reply:
d8b9db857278ac0944386363ad289f3328e79e2a